### ISSUE TITLE
Create sample...How to access premium content

### ISSUE THREAD

**Issue by BennyBaruba** [david-chambers/hackathon-handbook#24]
Create a web sample showing developers how to access premium content.

**Comment by david-chambers** [comment]
I can tackle this one.  This + possibly one for accessing BA data would be a great time-saver

**Comment by david-chambers** [comment]
I have been working on <a href=http://csc-dchambers7d/GitHub/arcgis-sdks/JavaScript/QueryPremiumElevationService/>this</a> (internal only) that will query the world elevation service <a href="http://www.arcgis.com/home/item.html?id=58a541efc59545e6b7137f961d7de883">here</a> While this may not be appropriate for the hackathon audience the pattern for other premium content is the same: 
1. Set up the Esri Resource Proxy or hosted ArcGIS Online Proxy
2. Add clientId and clientSecret from AGOL.
3. Setup application to use the proxy.  

My question: what premium service should we showcase for this sample? (Routing, Traffic, Tapestry, etc.)

**Comment by stewrouse** [comment]
I think Routing and/or Traffic.  Those are the services that would be most interesting to people.

**Comment by david-chambers** [comment]
@stewrouse Yeah I agree with you.  I was thinking this data would be cool to query as well: <a href="http://doc.arcgis.com/en/living-atlas/item/?itemId=94c2521892fb4544af7ba28228b9cf85&subType=demographics">Tapestry</a>
